mongo/datadog_checks/mongo/collectors/replica.py

```diff
@@ -88,8 +93,26 @@ def _report_replica_set_states(self, members, replset_name):
                 event_payload['host'] = self.hostname
             self.check.event(event_payload)
 
-    def collect(self, client):
-        db = client["admin"]
+    def get_replset_config(self, api):
```

Comment:
Behavioral change, likely should be removed from this PR.
For collecting the replica set config command, we used to run `replSetGetConfig` in all cases, which fails on arbiter nodes.
This new method first checks whether or not the node is an arbiter, and if so, creates an additional connection to the primary for running that command. The replica set configuration is the same on all nodes within a replica set.